README.md

```diff
@@ -20,7 +20,7 @@ The ASK SDK v2 for Node.js makes it easier for you to build highly engaging skil
 |[ask-sdk-s3-persistence-adapter](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-s3-persistence-adapter)| [![npm](https://img.shields.io/npm/v/ask-sdk-s3-persistence-adapter.svg)](https://www.npmjs.com/package/ask-sdk-s3-persistence-adapter) [![npm](https://img.shields.io/npm/dt/ask-sdk-s3-persistence-adapter.svg)](https://www.npmjs.com/package/ask-sdk-s3-persistence-adapter)|
 |[ask-sdk-v1adapter](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-v1adapter)|[![npm](https://img.shields.io/npm/v/ask-sdk-v1adapter.svg)](https://www.npmjs.com/package/ask-sdk-v1adapter) [![npm](https://img.shields.io/npm/dt/ask-sdk-v1adapter.svg)](https://www.npmjs.com/package/ask-sdk-v1adapter)|
 
-## Alexa Features Supported by SDK
+## Alexa Features supported by SDK
```

Comment:
This is intended to be in upper case.

Reply:
Makes sense too, I just compared it with [this readme](https://github.com/alexa/alexa-skills-kit-sdk-for-java/blob/2.0.x/Readme.md) and I thought lower case would fit better, but you got me

Reply:
Maybe we should update that readme instead. :D

Reply:
Yeah, I have a [Pull-Request there](https://github.com/alexa/alexa-skills-kit-sdk-for-java/pull/182), I'll include it